text/0225-configuration.md

```diff
@@ -0,0 +1,410 @@
+
+#### Python ParseAndValidateConfiguration example
+
+```python
+opentelemetry.ParseAndValidateConfiguration(
+    {
+        "scheme_version": "0.0.1",
+        "sdk": {
+            "resource": {
+                "attributes": {
+                    "service.name": "unknown_service",
+                }
+            },
+            "propagators": ["tracecontext", "baggage", "b3multi"],
+            "tracer_provider": {
+                "exporters": {
+                    "zipkin": {
+                        "endpoint": "http://localhost:9411/api/v2/spans",
+                        "timeout": 10000,
+                    },
+                    "otlp": {},
+                },
+                "span_processors": [
+                    {
+                        "name": "batch",
+                        "args": {
+                            "schedule_delay": 5000,
+                            "export_timeout": 30000,
+                            "max_queue_size": 2048,
+                            "max_export_batch_size": 512,
+                            "exporter": "zipkin",
+                        },
+                    }
+                ],
+            },
+            ...
+        }
+    })
+```
+
+### Go ParseAndValidateConfiguration example
+
+```go
+type config map[string]interface{} // added for convenience
+
+otel.ParseAndValidateConfiguration(config{
+    "sdk": config{
+      "resource": config{
+        "attributes": config{
+          "service.name": "unknown_service",
+        },
+      },
+      "propagators": []string{"tracecontext", "baggage", "b3multi"},
+      "tracer_provider": config{
+        "exporters": config{
+          "zipkin": config{
+            "endpoint": "http://localhost:9411/api/v2/spans",
+            "timeout":  10000,
+          },
+          "otlp": config{},
+        },
+        "span_processors": []config{
+          {
+            "name": "batch",
+            "args": config{
+              "schedule_delay":        5000,
+              "export_timeout":        30000,
+              "max_queue_size":        2048,
+              "max_export_batch_size": 512,
+              "exporter":              "zipkin",
+            },
+          },
+        },
+      },
+      ...
+    },
+  },
+)
+```
+
+### Configuration file
+
+The following demonstrates an example of a configuration file format (full example [here](https://github.com/MrAlias/otel-schema/blob/main/config.yaml)):
+
+```yaml
+# include version specification in configuration files to help with parsing and schema evolution.
+scheme_version: 0.1
+sdk:
+  # Disable the SDK for all signals.
+  #
+  # Boolean value. If "true", a no-op SDK implementation will be used for all telemetry
+  # signals. Any other value or absence of the variable will have no effect and the SDK
+  # will remain enabled. This setting has no effect on propagators configured through
+  # the OTEL_PROPAGATORS variable.
+  #
+  # Environment variable: OTEL_SDK_DISABLED
```

Comment:
Is there a consistent mapping between env variable names and keys in the config file?

Reply:
By consistent I assume you mean that you can infer the equivalent position of an environment variable in the schema based on its name. Something like `OTEL_SDK_DISABLED` maps to:
```
otel:
  sdk:
    disabled: ...
```
There is not a consistent mapping.

Reply:
+1 for no consistent mapping. The environment variables are named inconsistently and this is a perfect time to rectify that mistake

Reply:
Added a note that there is no consistent mapping between environment variables and keys in the config.